Entities/Ingame/ClubFinances.cs

```diff
@@ -74,9 +74,17 @@ public ClubFinances (int memoryAddress, ArraySegment<byte> originalBytes, IVersi
 		}
 
 		public int WeeklyWageBudget {
-			get {
-				return PropertyInvoker.Get<Int32> (ClubFinancesOffsets.WeeklyWageBudget, OriginalBytes, MemoryAddress, DatabaseMode);
-			}
+			get {
+                int value = PropertyInvoker.Get<Int32>(ClubFinancesOffsets.WeeklyWageBudget, OriginalBytes, MemoryAddress, DatabaseMode);
+                int subValue = PropertyInvoker.Get<Int32>(0x4C, OriginalBytes, MemoryAddress, DatabaseMode);
+
+                if (value == 0 || subValue == 0 || subValue < 0 || subValue > 10000)
+                {
+                    return 0;
+                }
+                value = value + subValue;
+                return value;
+            }
 		}
```

Comment:
When I tested it the old address was off by a few 100/1000 (doesn't seem the same for every player), when I added a breakpoint to the address in OllyDbg I saw that the address is encrypted by that value at 0x4C now. Here's a screenshot - http://i.imgur.com/gnwpVvd.png . Let me know if there's a better way to do it. 

Yeah sorry I forgot about compatibility with other versions, I'll go through and fix that tomorrow :)